openedx/core/djangoapps/safe_sessions/middleware.py

```diff
@@ -390,6 +419,14 @@ def process_response(self, request, response):
             except SafeCookieError:
                 _mark_cookie_for_deletion(request)
 
+        if destroy_session:
+            # Destroy session in DB.
+            request.session.flush()
```

Comment:
For reference: https://github.com/django/django/blob/3.2.10/django/contrib/sessions/backends/base.py#L329